### DISCUSSION TITLE
Unsupported RabbitMQ version (< 3.10.0)

### DISCUSSION THREAD

**Discussion by javi-on** [rabbitmq/rabbitmq-server#10035] [Questions] [answered]
Hi all new here, we are currently running 3.8.x version of rabbitmq and we are trying to test the upgrade before upgrading, but since we are with a lot of work right now, the idea of asking the community if there was some sort of upgrading window when either the service will stop working with that version or a compulsory upgrade is done, is there something like this? is there any schedule for this?

Thanks in advance

**Comment by lukebakken** [comment, marked as answer]
You can run version `3.8.x` as long as you like, but you will not get support from the RabbitMQ team.

**Reply by javi-on** [reply]
Thanks i was expecting this, but good to know anyways!
